ArcadiaParties-SPA/src/app/user/models/User.ts

```diff
@@ -1,8 +1,10 @@
+import { Department } from 'src/app/department/models/Department';
```

Comment:
This file's name should also be in camel-case.

Reply:
This file too

Reply:
Here too **lower**CamelCase.

Reply:
Files have been renamed